addons/event/models/event_registration.py

```diff
@@ -83,25 +98,34 @@ def _prepare_attendee_values(self, registration):
         data.update({key: value for key, value in registration.items() if key in self._fields})
         return data
 
+    def name_get(self):
+        ret_list = []
+        for registration in self:
+            if not registration.partner_id or not registration.partner_id.name:
```

Comment:
Well code is quite obscure here. I suppose purpose is to see both partner name and name, as you could have several registrations linked to a given partner.

Can't you simply do
```
# get both partner name and name, if defined, and remove duplicates
parts = list(set([p for p in registration.partner_id.name, registration.name]))
name = ', '.join(part)
```
Not sure it is completely the same, to check, but at least it is simpler to read :D .

Reply:
@tde-banana-odoo 
If the partner name and the registration name are not the same, we want to have "Partner, registration".
If I use a set, the order will not be kept :/

Reply:
Well you got the idea ...